assets/prefabs/potion/DivergencePotion.prefab

```diff
@@ -0,0 +1,33 @@
+{
+    "parent": "engine:iconItem",
+    "Item": {
+        "icon": "Potions:DivergencePotionBottle",
```

Comment:
While this seems to match the file name I don't see the the texture get used in-game - instead I see the "not found" texture. Might be from the image having `.PNG` as an extension rather than `.png` ? Try out the change and make sure to test this in-game by opening the console (with F1) and using `give DivergencePotion` to get the potion. You should see the valid potion texture when it works :-)